app/views/modules/_avalon_search_form.html.erb

```diff
@@ -20,14 +20,26 @@ Unless required by applicable law or agreed to in writing, software distributed
   <input id="search_field" name="search_field" type="hidden" value="all_fields">
   <div>
     <label class="sr-only" for="searchField">Search</label>
-    <div class="input-group input-group-lg global-search-wrapper">
+    <div class="input-group global-search-wrapper">
       <input class="search-query q form-control global-search-input" id="searchField" name="q"
         placeholder="Search this site" type="text" value="">
       <span class="input-group-btn">
-        <button id="global-search-submit" type="submit" class="btn btn-primary">
+        <button id="global-search-submit" type="submit" class="btn">
           <i class="fa fa-search icon-white"></i>
         </button>
       </span>
     </div>
   </div>
 </form>
+
+<% content_for :page_scripts do %>
```

Comment:
What happens if a user changes the screen size?  This might not get picked up.  Is there any way to handle this exclusively through CSS?

Reply:
Also, looks like this code gets doubled up?  If so, means there are 2 DOM elements with the same `id`, which will cause issues and technically is invalid HTML.

![image](https://user-images.githubusercontent.com/3020266/69081197-1a430180-0a03-11ea-877b-66a8a5bb6a1a.png)

Reply:
Good catch!! Yes, only the button changes when screen size changes not the input class. Let me see whether there's a way to do this with CSS.
This HTML component is rendered in 2 different places, for mobile and desktop views. And only one is visible at a given time.

Reply:
Ah, this is probably the source of the failing capybara tests too:
`rspec ./spec/features/capybara_navigation_spec.rb:73`